### ISSUE TITLE
No viz, just directory listing

### ISSUE THREAD

**Issue by madchap** [sebastientaggart/docker-cloudmapper#5]
Hi,

Probably just incompetence on my end, but after was seems a successful collection and prep, I only have a data.json file within the web folder, and only directory listing is presented on port :8000. No cyto interactive diagram.

Is there something I am doing wrong or missing?

Thank you.

**Comment by tmestdagh** [comment]
I'm experiencing the same thing.  Did you every find a solution @madchap ?

**Comment by madchap** [comment]
Hi @tmestdagh 

No, stuff came up and I did not pursue. I would still be interested if there is an answer though :)

Cheers.

**Comment by madchap** [comment]
I am not mistaken though, I think @0xdabbad00 is planning to work again on a Docker image upstream.

**Comment by 0xdabbad00** [comment]
Yes, I ticketed https://github.com/duo-labs/cloudmapper/issues/287 

I don't have any timeline though for when I'll get to it.

**Comment by tmestdagh** [comment]
Thanks @madchap , I'll keep an eye on it.   @0xdabbad00  Is there a branch uploaded with the work you already have done for this docker image?  I would like to help out a bit.

**Comment by 0xdabbad00** [comment]
@tmestdagh No work has been started for it.  There had been a few PRs for Docker support to CloudMapper though:
- https://github.com/duo-labs/cloudmapper/pull/219
- https://github.com/duo-labs/cloudmapper/pull/221
- https://github.com/duo-labs/cloudmapper/pull/54
- https://github.com/duo-labs/cloudmapper/pull/27

**Comment by syucream** [comment]
I have same issue. Don't anyone have any workaround?

**Comment by syucream** [comment]
I found the cause is mounting empty web/ directory. I made a patch for me to avoid the using volume:
https://github.com/syucream/docker-cloudmapper/commit/f2f1b5a5bb308828c9d6c4f3c0da3d358109423d

**Comment by fluidesign** [comment]
The root cause of the issue the fact that the docker-compose using VOLUME pointing to the web folder of the local host, which is empty.
The solution as mentioned above is to remove the VOLUME from the docker-compose file point to the web folder.
